tests/resources/vocabularies/test_awards_vocabulary.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Award resource tests."""
+
+import pytest
+from invenio_records_resources.proxies import current_service_registry
+from invenio_vocabularies.contrib.awards.api import Award
+
+
+@pytest.fixture(scope="module")
+def awards_service():
+    return current_service_registry.get("rdm-awards")
+
+
+@pytest.fixture()
+def example_award(
+    app, db, es_clear, superuser_identity, awards_service
+):
+    """Example award."""
+    data = {
+        "pid": "755021",
+        "identifiers": [
+            {
+                "identifier": "https://cordis.europa.eu/project/id/755021",
+                "scheme": "url"
+            }
+        ],
+        "number": "755021",
+        "title": {
+            "en": "Personalised Treatment For Cystic Fibrosis Patients With \
+                Ultra-rare CFTR Mutations (and beyond)",
+        },
+        "funder": {
+            # FIXME: add appropriate value when funders has been integrated
```

Comment:
might be a good idea to tests also the dereferencing, i.e. save an award with only a `funder.id` and then query both the item endpoint (`/award/1231231`) and the search to test the dereferencing works at db and es level. Similar tests are done at invenio-vocabulares level (both in awards and names)